submit/submit.go

```diff
@@ -178,8 +178,8 @@ func Main() {
 			return
 		}
 	}
-	var ebsCmd [4]string
-	ebsCmd[0] = `cleanup_volume() { echo "batchit: No volumes to clean up"; }`
+	cleanupDefault := `cleanup_volume() { echo "batchit: No volumes to clean up"; }`
```

Comment:
This look better?

Reply:
yes. just make the default be the empty function with no echo.